src/encoded/static/components/region_search.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import url from 'url';
-import { BrowserSelector } from './objectutils';
```

Comment:
I think that `BrowserSelector` from `objectutils.js` isn't used anywhere else, so the component could probably be deleted.

Reply:
I already deleted `BrowserSelector` from objectutils.js last release I think. It only exists in vis_defines.js now.

Reply:
Got it, sounds good